docs/intro/tutorial.rst

```diff
@@ -18,11 +18,12 @@ This tutorial will walk you through these tasks:
 4. Changing spider to recursively follow links
 5. Using spider arguments
 
-Scrapy is written in Python_. If you're new to the language you might want to
-start by getting an idea of what the language is like, to get the most out of
+Scrapy is written in Python_. If you're new to the language, you might want to
+start by getting an idea of what the language is like to get the most out of
 Scrapy.
```

Comment:
I need an explanation for this change beyond the fact that a grammar tool suggested it. I am not opposed to the new comma, but removing the other one seems wrong to me.

Reply:
A comma should follow a subordinate clause; in this case, “If you’re new to the language”. When an independent clause is followed by a subordinate clause, you don’t need to use a comma, making the one between “like” and “to” incorrect.

Reply:
What if we replace the entire sentence with a shorter one?

> Scrapy is written in Python. The more you learn about Python, the more you can get out of Scrapy.

Reply:
Yes, I think that should work.